add-domain

```diff
@@ -522,3 +522,15 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+www.roblox.com.es
+newappmail.discoverynevers.com
```

Comment:
Seems to have been solved

![image](https://user-images.githubusercontent.com/44526987/126139350-71112c3f-0043-4177-af56-dcf195ab7230.png)